openquake/server/tests/data/vulnerability_model_invalid.xml

```diff
@@ -0,0 +1,59 @@
+            </discreteVulnerability>
+			<discreteVulnerability vulnerabilityFunctionID="PCR-SLSB-B" probabilisticDistribution="LN">
+                   <lossRatio>0.00077	0.00311	0.00767	0.02038	0.04843	0.09576	0.14684	0.19792	0.23070	0.27795	0.31619	0.36601	0.42323	0.49028	0.55603	0.61983	0.68054	0.73489	0.78495	0.82929	0.86556	0.89732	0.92350	0.94306	0.95933	0.97175	0.98020	0.98687	0.99154	0.99440	0.99654	0.99794	0.99875	0.99930	0.99957	0.99978	0.99990	0.99990	0.99996	1	1	1	1	1	1	1	1	1	1	1</lossRatio>
+                  <coefficientsVariation>0.00092	0.00373	0.00920	0.02372	0.05518	0.09870	0.14456	0.19043	0.21275	0.24081	0.25912	0.27668	0.29081	0.29883	0.29333	0.28087	0.26004	0.23197	0.20163	0.16937	0.13884	0.11028	0.08463	0.06422	0.04675	0.03292	0.02331	0.01558	0.00997	0.00665	0.00418	0.00258	0.00151	0.00082	0.00062	0.00024	0	0	0	0	0	0	0	0	0	0	0	0	0	0</coefficientsVariation>
+            </discreteVulnerability>
+			<discreteVulnerability vulnerabilityFunctionID="W-FLFB-2" probabilisticDistribution="LN">
+                   <lossRatio>0.00055	0.00236	0.01031	0.02451	0.04527	0.07354	0.11648	0.17262	0.23886	0.31383	0.39681	0.48321	0.56895	0.65171	0.72827	0.79343	0.84885	0.89443	0.93026	0.95403	0.97128	0.98325	0.99103	0.99508	0.99742	0.99875	0.99945	0.99975	0.99986	0.99994	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1</lossRatio>
+                  <coefficientsVariation>0.00068	0.00289	0.01229	0.02854	0.05153	0.08160	0.12309	0.16962	0.21557	0.25689	0.28600	0.29621	0.29069	0.27055	0.23686	0.19503	0.15277	0.11283	0.07782	0.05238	0.03334	0.01974	0.01071	0.00590	0.00303	0.00138	0.00056	0.00034	0.00011	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0</coefficientsVariation>
+            </discreteVulnerability>
+			<discreteVulnerability vulnerabilityFunctionID="W-SLFB-1" probabilisticDistribution="LN">
+                   <lossRatio>0.00028498	0.0023533	0.0090565	0.022609	0.044559	0.076123	0.11779	0.16957	0.23059	0.29953	0.37414	0.45194	0.53011	0.60612	0.67766	0.74264	0.79988	0.84854	0.88861	0.92046	0.94495	0.96308	0.97604	0.98502	0.99095	0.99472	0.99706	0.99838	0.99914	0.99957	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1	1</lossRatio>
+                  <coefficientsVariation>0.00037997	0.0028769	0.01069	0.026463	0.050895	0.084018	0.12433	0.1688	0.21288	0.25159	0.28065	0.29646	0.2979	0.2854	0.26116	0.22849	0.1913	0.15358	0.11837	0.087579	0.062307	0.042706	0.02806	0.017718	0.010778	0.0062203	0.0035689	0.0020238	0.0010342	0.00052825	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0</coefficientsVariation>
+            </discreteVulnerability>
+        </discreteVulnerabilitySet>
+    </vulnerabilityModel>
+</nrml>
```

Comment:
Please reduce the size of this file to maximum 10 lines. Also, try to use negative coefficients to make this invalid.